### PR TITLE
feat: games

### LINKED ISSUE #34: gaming mode
N/A

### PR DESCRIPTION
**Pull request by TitusKirch**

### CODE REVIEW

**Comment by TitusKirch** [comment]
Closed #34

File: extensions/games.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        await ctx.author.add_roles(*roles)
+
+        # create output embeds
+        embed_success = discord.Embed(
+            colour=discord.Colour.green(),
+            title='Erfolgreich'
+        )
+        embed_has = discord.Embed(
+            colour=discord.Colour.blue(),
+            title='Schon in Gruppen'
```
Comment (Rec0gnice) [inline extensions/games.py]: double title prefix "Schon"?

File: extensions/meta/games.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "name": "games",
+    "description": "The Cake Is A Lie.",
```
Comment (Rec0gnice) [inline extensions/meta/games.json:3]: I like it

File: extensions/games.py (hunk 1 above)
Comment (TitusKirch) [reply extensions/games.py]: ?

Comment (TitusKirch) [reply extensions/games.py:66]: ?

**Review by Rec0gnice — APPROVED** [review]

**Pull request merged**